onnxruntime/test/providers/cpu/activation/activation_op_test.h

```diff
@@ -74,6 +74,10 @@ inline void TestActivationOp(const char* szOp, const std::vector<std::vector<T>>
       test.SetOutputTolerance(0.0001f);
     }
 
+    if (strcmp(szOp, "Gelu") == 0) {
+      test.SetOutputTolerance(0.00015f);
```

Comment:
This affects all EP's. Should we bracket it within an `#if defined(USE_DML)` like line 61 for DNNL? Otherwise LGTM.